components/CreateAlertModal.tsx

```diff
@@ -258,6 +259,21 @@ const CreateAlertModal: FunctionComponent<CreateAlertModalProps> = ({
     actions.loadAlerts(mangoAccount.publicKey)
   }, [])
 
+  // Delete notifi Alerts that have fired
+  useEffect(() => {
+    const firedAlert = data?.alerts.find((item) => {
+      const filterOptions = item.filterOptions ?? '{}'
+      const parsedOptions = JSON.parse(filterOptions)
+      return parsedOptions.delayProcessingUntil === '12/31/9999 23:59:59'
+    })
+
+    if (firedAlert !== undefined && firedAlert.id !== null) {
+      deleteAlert({ alertId: firedAlert.id }).then(() => {
```

Comment:
I think we delete the alert from mango DB after sending the alert to notifi. So not quite sure why we would need the client to delete it again.

Reply:
Oh, we should remove that behavior from Mango Alert server

Reply:
Oh, i think this was because there was confusion around `deleteAlert` vs. `actions.deleteAlert` -- this line actually delets the *notifi* alert.